<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>763</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1106</width>
    <height>763</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1106</width>
    <height>763</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MultiTAC</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>ui/handIcon.png</normaloff>ui/handIcon.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonIconOnly</enum>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QFrame" name="System">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <widget class="QPushButton" name="ConnectButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>560</y>
         <width>221</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLabel" name="WelcomeText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>231</width>
         <height>81</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>16</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Welcome to MultiTAC User Interface</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="WelcomeText2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>70</y>
         <width>211</width>
         <height>251</height>
        </rect>
       </property>
       <property name="text">
        <string>
What is MultiTAC?

Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.
</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignJustify|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="SystemStatusText">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>370</y>
         <width>231</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>System Status</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>350</y>
         <width>251</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>420</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_25">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Connected</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="connectedBox">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::StrongFocus</enum>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::DefaultContextMenu</enum>
          </property>
          <property name="acceptDrops">
           <bool>false</bool>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="iconSize">
           <size>
            <width>80</width>
            <height>80</height>
           </size>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>460</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_26">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Powered</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="poweredBox">
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="Experiment">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QFrame" name="Setup">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <widget class="QLabel" name="label_15">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>381</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Experiment Setup</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>381</width>
            <height>286</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="10" column="2">
            <widget class="QPushButton" name="BeginButton">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Go!</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QLabel" name="label_16">
             <property name="text">
              <string>Distance</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="label_17">
             <property name="text">
              <string>cm</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Body Site</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Velocities</string>
             </property>
            </widget>
           </item>
           <item row="2" column="3">
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>cm/s</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLabel" name="label_20">
             <property name="text">
              <string>Repetitions per velocity</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="VelocitiesInput">
             <property name="placeholderText">
              <string>1;3;10;...</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="BodySite">
             <item>
              <property name="text">
               <string>Forearm</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Upper Back</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Lower Back</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Thigh</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Calf</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Maximum time for VAS response</string>
             </property>
            </widget>
           </item>
           <item row="7" column="3">
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>s</string>
             </property>
            </widget>
           </item>
           <item row="8" column="3">
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>s</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="4">
            <widget class="QLabel" name="label_14">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>InterTrial time interval</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QLabel" name="label_6">
             <property name="lineWidth">
              <number>1</number>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string>Brush Movement</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="MovementDirection">
             <item>
              <property name="text">
               <string>Linear</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Circular</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Oval</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="11" column="0" colspan="4">
            <widget class="QLabel" name="label_18">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QLineEdit" name="distanceInput"/>
           </item>
           <item row="6" column="2">
            <widget class="QLineEdit" name="repetitionsInput"/>
           </item>
           <item row="7" column="2">
            <widget class="QLineEdit" name="VastimeInput"/>
           </item>
           <item row="8" column="2">
            <widget class="QLineEdit" name="intertrialIntput"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFrame" name="Subject">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>381</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Subject Information</string>
          </property>
         </widget>
         <widget class="QPushButton" name="ResetButton">
          <property name="geometry">
           <rect>
            <x>320</x>
            <y>270</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>11</x>
            <y>40</y>
            <width>381</width>
            <height>111</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="1">
            <widget class="QLineEdit" name="ageInput"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="subjectInput"/>
           </item>
           <item row="2" column="1">
            <widget class="QComboBox" name="genderInput">
             <item>
              <property name="text">
               <string>Woman</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Man</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Transgender</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Non-binary/non-conforming</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Prefer not to respond</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Handedness</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QComboBox" name="handInput">
             <property name="currentText">
              <string>Left</string>
             </property>
             <item>
              <property name="text">
               <string>Left</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Right </string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Ambidextrous</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Subject</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_19">
             <property name="text">
              <string>years old</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Age</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Gender Identity</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QTabWidget" name="Output">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="Page1">
          <property name="accessibleName">
           <string>Page1</string>
          </property>
          <attribute name="title">
           <string>Console Ouput</string>
          </attribute>
          <widget class="QProgressBar" name="progressBar">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>290</y>
             <width>781</width>
             <height>23</height>
            </rect>
           </property>
           <property name="value">
            <number>0</number>
           </property>
           <property name="textVisible">
            <bool>false</bool>
           </property>
          </widget>
          <widget class="QPlainTextEdit" name="ConsoleOutput">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>10</y>
             <width>781</width>
             <height>271</height>
            </rect>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Page2">
          <attribute name="title">
           <string>Demo</string>
          </attribute>
          <widget class="QPushButton" name="demoButton">
           <property name="geometry">
            <rect>
             <x>320</x>
             <y>270</y>
             <width>471</width>
             <height>23</height>
            </rect>
           </property>
           <property name="text">
            <string>Execute demo!</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Page3">
          <attribute name="title">
           <string>About</string>
          </attribute>
          <widget class="QLabel" name="AboutText">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>20</y>
             <width>451</width>
             <height>121</height>
            </rect>
           </property>
           <property name="text">
            <string>Created By: Edgardo Silva - Universidad de Valparaíso
Contact: edgardo.silva@uv.cl

MultiTAC Project: Edgardo Silva; Grace Whitaker; Alejandro Weinstein; Wael El-Deredy; Francis McGlone

Selected publications: </string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_21">
           <property name="geometry">
            <rect>
             <x>670</x>
             <y>90</y>
             <width>121</width>
             <height>91</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>thumbnail.svg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_22">
           <property name="geometry">
            <rect>
             <x>520</x>
             <y>90</y>
             <width>121</width>
             <height>91</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>thumbnail.svg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_23">
           <property name="geometry">
            <rect>
             <x>520</x>
             <y>210</y>
             <width>121</width>
             <height>91</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>thumbnail.svg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_24">
           <property name="geometry">
            <rect>
             <x>670</x>
             <y>210</y>
             <width>121</width>
             <height>91</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>thumbnail.svg</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>21</height>
    </rect>
   </property>
   <property name="defaultUp">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
